content/kubermatic/v2.18/architecture/known_issues/_index.en.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Known Issues"
+date = 2022-07-20T12:22:15+02:00
+weight = 1
+
++++
+
+## Overview
+
+This page documents the list of known issues description and possible work arounds/solutions.
+
+### 1. OIDC user authentication issue
+
+**Problem** 
+
+[OIDC](https://docs.kubermatic.com/kubermatic/v2.20/tutorials_howtos/oidc_provider_configuration/share-_clusters_via_delegated_oidc_authentication/) user is denied to access the user cluster in the KKP with K8s version of 1.20 and below. Refer the github issue [Bug: OIDC authentication...](https://github.com/kubermatic/kubermatic/issues/9908) for detailed problem description. Example logs look like below,
+
+Kubectl output
+
+```k get nodes
+error: You must be logged in to the server (Unauthorized)
+```
+
+API server logs
+
+```
+2022-05-26T11:46:11.269134597Z stderr F E0526 11:46:11.267368       1 authentication.go:63] "Unable to authenticate the request" err="[invalid bearer token, oidc: authenticator not initialized]"
+2022-05-26 13:46:11	
+2022-05-26T11:46:11.200645694Z stderr F E0526 11:46:11.200494       1 authentication.go:63] "Unable to authenticate the request" err="[invalid bearer token, oidc: authenticator not initialized]"
+2022-05-26 13:46:10	
+2022-05-26T11:46:10.282230799Z stderr F E0526 11:46:10.282080       1 oidc.go:224] oidc authenticator: initializing plugin: Get "https://run.lab.kubermatic.io/dex/.well-known/openid-configuration": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
```

Comment:
do not use actual domain (`run.lab.kubermatic.io`), use something like `your-kkp.domain` or something similar